crates/key-value-azure/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = { workspace = true }
 
 [dependencies]
 anyhow = "1"
-azure_data_cosmos = "0.11.0"
+azure_data_cosmos = { git = "https://github.com/azure/azure-sdk-for-rust.git", rev = "8c4caa251c3903d5eae848b41bb1d02a4d65231c" }
+azure_identity = { git = "https://github.com/azure/azure-sdk-for-rust.git", rev = "8c4caa251c3903d5eae848b41bb1d02a4d65231c" }
```

Comment:
These are not yet part of a release, right?

Reply:
They are not. Also, I'm not clear on when they would be.